s64da_benchmark_toolkit/reporting.py

```diff
@@ -96,7 +96,7 @@ def run_report(self, reporting_queue):
         netdata_config = self.config.get('netdata')
         if netdata_config:
             netdata = Netdata(netdata_config)
-            if self.netdata_output_file:
+            if self.netdata_output_file and netdata.is_netdata_set_and_running:
```

Comment:
This should be:
`and self.is_netdata_set_and_running():`

Reply:
Also, IMO it'd be better to move `netdata_output_file` into `Netdata`, then you have it encapsulated.